docs/faq.md

```diff
@@ -98,6 +100,13 @@ Generally, yes, with two exceptions:
 - hiyaCFW will only work on the system it was set up for
 - Even if you are using nds-bootstrap or a flashcard, friend codes in online NDS games will be reset when attempting to go online using a different console
 
+## How do I switch to a new SD card after setting up homebrew?
+Format your new SD card using the [SD Card Setup](sd-card-setup.html) instructions, then simply move your data from the old SD card to the new one.
+
+## Can I still use my system normally without the SD card inserted after setting up homebrew?
+Yes. If you did not install Unlaunch, your system will remain completely unmodified.
+- If you *did* install Unlaunch, you may need to [configure Unlaunch](installing-unlaunch.html#section-iii-post-unlaunch-configuration) to automatically boot the original DSi Menu under specified conditions
```

Comment:
I think the bullet point seems a bit out of place here, I think would be better as just:
```suggestion
Yes. If you did not install Unlaunch, your system will remain completely unmodified. If you *did* install Unlaunch, you may need to [configure Unlaunch](installing-unlaunch.html#section-iii-post-unlaunch-configuration) to automatically boot the original DSi Menu under specified conditions.
```